internals/webpack/webpack.base.babel.js

```diff
@@ -70,7 +70,7 @@ module.exports = options => {
               loader: 'svg-url-loader',
               options: {
                 // Inline files smaller than 10 kB
```

Comment:
comment not needed anymore?

Reply:
It was here earlier, I think It is enough informative and there is no necessity to remove it